kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -871,6 +871,52 @@ class ResidualBasedBlockBuilderAndSolver
         }
     }
 
+    void ApplyConstraints(

```

Comment:
what is the difference with this? is it only moved within the file?

Reply:
it is moved to `public`